conan/tools/cmake/cmakedeps/cmakedeps.py

```diff
@@ -42,7 +42,7 @@ def generate(self):
         # Current directory is the generators_folder
         generator_files = self.content
         for generator_file, content in generator_files.items():
-            save(generator_file, content)
+            save(self._conanfile, generator_file, content)
```

Comment:
Should we change the ``save()`` and ``load()`` usages to use the ``from conan.tools.files`` ones?

Reply:
My question goes the other way around. Any reason why we would not want to?

Reply:
The current implementation of internal ones feature a ``save(..., only_if_modified, )`` which in theory is useful for generators, but in practice not used anymore, that the ``tools.files`` ones are missing.

But otherwise, yes, I think trying to make the ``conan.tools`` as self-contained as possible is good. There will be some exceptions for our own internal deep things that should go to ``from conan.internal`` to clearly expose and differentiate that detail.
The problem with the current status is that it is nearly unnoticeable that you are using things from a "private/internal" scope that you shouldn't be using